frontend/src/components/nearbyPage/NearbyPage.jsx

```diff
@@ -1,12 +1,30 @@
 import Header from '../header/Header'
+import CardGridEvent from '../CardGridEvent'
+import { useState, useEffect } from 'react'
 
-function App() {
+function NearbyPage() {
+    const url = import.meta.env.VITE_URL
+    const userData = JSON.parse(localStorage.getItem('user'))
```

Comment:
It should be user state, instead of localStorage here. I have pointed this out in the past PR.

Reply:
Yes, I haven't figured out how to use useContext for that.
I'll fix it today

Reply:
I'll fix it today

Reply:
Done